pkg/compute/compute_integration_test.go

```diff
@@ -955,6 +1056,49 @@ func makeBuildEnvironment(t *testing.T, fastlyManifestContent, cargoManifestCont
 	return rootdir
 }
 
+func makeAssemblyScriptBuildEnvironment(t *testing.T, fastlyManifestContent string) (rootdir string) {
+	t.Helper()
+
+	p := make([]byte, 8)
+	n, err := rand.Read(p)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rootdir = filepath.Join(
+		os.TempDir(),
+		fmt.Sprintf("fastly-build-%x", p[:n]),
+	)
```

Comment:
This was my dumb code initially 🤦 which can be replaced with

```suggestion
	rootdir = ioutil.TempDir("", "fastly-build-")
```

Reply:
Nice. Have implemented in https://github.com/fastly/cli/pull/160/commits/b6776d3b6105ed6df7cc9e03c2d421392933d602